build_binary.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -e
+
+yum install rpm-build rpmdevtools python-devel -y
+yum groupinstall "Development Tools" -y
+
+rpmdev-setuptree
+
+pip install git+https://github.com/thoughtworksinc/pex#egg=pex
```

Comment:
We had to patch PEX sometime back, to persist permissions on files bundled inside PEX. 
We have raised the pull request to PEX to merge this change, its on their list. Till then we have to use this patched PEX.
Can move this to MS repo?